config/develop/rna-seq-reprocessing-output-bucket.yaml

```diff
@@ -21,6 +21,7 @@ parameters:
     - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
     - 'arn:aws:iam::563295687221:user/tess.thyer@sagebase.org'
     - 'arn:aws:iam::563295687221:user/william.poehlman@sagebase.org'
+    - 'arn:aws:iam::563295687221:role/rna-seq-reprocessing-instance-role-v00-ClusterRole-1QAEKIKZY9I2L'
```

Comment:
You've added a dependency on the role in [rna-seq-reprocessing-instance-role-v001.yaml](https://github.com/Sage-Bionetworks/toil-cluster-infra/blob/develop/config/develop/rna-seq-reprocessing-instance-role-v001.yaml).  That probably means you should add:
```
dependencies:
  - develop/rna-seq-reprocessing-instance-role-v001.yaml
```